lib/ibandit/iban.rb

```diff
@@ -52,6 +52,18 @@ def swift_national_id
       national_id.slice(0, structure[:national_id_length])
     end
 
+    def account_number
+      return @account_number unless country_code == "NZ"
```

Comment:
Special casing here is not ideal, however, we were faced with two options:
1. Attempt a relatively large refactor
2. Special case

Given the fact that:
1. Internally we are currently looking at alternatives to Ibandit so we're not ready to invest in a refactor until we're sure it's the right thing to do
2. A large refactor open us up to regressions in critical flows

We believe that for now, this is the simplest approach we can take which avoids us re-writing a significant chunk of this gem.